### ISSUE TITLE
Feature request: Element Exists ... keywords with boolean result value

### ISSUE THREAD

**Issue by bartkl** [robotframework/SeleniumLibrary#1313]
Hello,

As an avid user of Robot Framework and SeleniumLibrary, I often find myself checking if an element exists using the following pattern:

```robot
    ${kw on fail}=  Register keyword to run on failure  NONE
    ${element is on page}=  Run keyword and return status
    ...  Wait until page contains element  ${locator}
    Register keyword to run on failure  ${kw on fail}
```
After that, I can use the boolean `${element is on page}` to do some or other thing.

It's especially annoying to have to disable and re-enable the screenshot capturing before and after the check. I often forget about this which leads to many unwanted images in my results directory.

In my mind it would make a lot of sense to have a `Element Exists On Page` keyword (and similar ones) that returns a boolean value describing whether the element is present or not, rather than asserting its presence like `Page should contain element` does, or wait for it to necessarily appear like `Wait until page contains element` does. Note that this/these new keyword(s) would not make any screenshots: they replace the tedious pattern I described above.

I hope this feature request makes sense.

Thanks,
Bart

**Comment by aaltat** [comment]
There is a similar request, but for different keyword, in #1127 issue. I like the idea presented in the #1127 that existing keywords could have parameter where it could be controlled does the `Page Should Contain...` raise an exception or return a boolean (based on did it succeed or failed.)

For consistency reasons, it should be done for all keywords in a single release. But no-one has seen the feature important enough to provide a PR.

**Comment by bartkl** [comment]
I have given that thought as well, and I appreciate the generality which is definitely desirable. I also understand that might not be worth the effort though.

However, I think my request significantly differs from the one you compare it with because of the temporary disabling of screenshots. The pattern I described is the one Pekka suggested in #1127 but with the temporary disabling of screenshots. Whereas Pekka's example is reasonable, the screenshot disabling I added makes it very cumbersome.
